CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -431,6 +431,20 @@ function PlayerHospital:advisePriceLevelImpact(judgment, name)
   self.world.ui.adviser:say(message)
 end
 
+--! Makes the raise request for a staff member
+--!param amount (num) the requested raise increase
+--!param staff (table) the staff member
+function PlayerHospital:makeRaiseRequest(amount, staff)
+  -- Show advice if it is the first time the player has experienced
+  -- a staff member requesting a raise.
+  -- Only show the help if the player is playing the campaign.
+  if self.has_seen_pay_rise and tonumber(self.world.map.level_number) then
```

Comment:
Should be `if not self.has_seen_pay_rise` ?

Reply:
Yes